src/parsing/merchants.spec.ts

```diff
@@ -0,0 +1,138 @@
+import type { Network } from "./types"
+import { convertMerchantQRToLightningAddress } from "./merchants"
+
+describe("convertMerchantQRToLightningAddress", () => {
+  // Test cases for valid QR contents and networks
+  test.each([
+    {
+      description: "PicknPay EMV QR code on mainnet",
+      qrContent:
+        "00020129530023za.co.electrum.picknpay0122RD2HAK3KTI53EC/confirm520458125303710540115802ZA5916cryptoqrtestscan6002CT63049BE2",
```

Comment:
These are moneybadger generated test QR data, identifiable by the presence of the string "cryptoqrtestscan".

It would be good also to include in the tests actual merchant generated QR data.

Here is an example of a merchant generated code in pnp's production environment:
`00020126260008za.co.mp0110248723666427530023za.co.electrum.picknpay0122ydgKJviKSomaVw0297RaZw5303710540571.406304CE9C`

Here is an example of a merchant generated code in pnp's test environment:
`00020126260008za.co.mp0110628654976427530023za.co.electrum.picknpay0122a/r4RBWjSNGflZtjFg4VJQ530371054041.2363044A53`